tests/sles4sap/wizard_hana_install.pm

```diff
@@ -43,6 +44,14 @@ sub run {
     # Add host's IP to /etc/hosts
     $self->add_hostname_to_hosts;
 
+    # Install libopenssl1_0_0 for older (<SPS03) HANA versions on SLE15+
+    my $hana_version = $path =~ s/.*\/SPS([0-9]+)rev[0-9]\/.*/$1/r;
+    if (is_sle('15+') && ($hana_version <= 2)) {
+        # The old libopenssl is in Legacy Module
+        add_suseconnect_product('sle-module-legacy');
+        zypper_call('in libopenssl1_0_0');
```

Comment:
This code is the same in both modules. It's a good candidate to be transformed into a method in `lib/sles4sap`.

Reply:
Done.